### PR TITLE
Build/performance

### PR DESCRIPTION
**Pull request by raop155**
# Description

- Compress assets
- "Improve" chart styling

### CODE REVIEW

File: src/components/molecules/CrossChainChart/Chart.tsx [hunk 1]
```diff
@@ -48,6 +47,7 @@ export const Chart = ({ data, selectedType }: Props) => {
   const size = useWindowSize();
   const [isDesktop, setIsDesktop] = useState(size.width >= BREAKPOINTS.desktop);
 
+  const minHeight = 42;
```
Comment (valentinoConti) [inline src/components/molecules/CrossChainChart/Chart.tsx:50]: revert this, there is a min height for every different breakpoint on the scss 🙏🏼

Comment (raop155) [reply src/components/molecules/CrossChainChart/Chart.tsx:50]: I will revert this (min-heights), you are right, but all looks too tight. there should be padding/whitespace.
image

Leave to you then, that you know more how should work properly.

File: src/index.html [hunk 2]
```diff
@@ -6,8 +6,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <meta name="theme-color" content="#010014" />
-  <link rel="icon" href="favicon_dark.png" media="(prefers-color-scheme: light)" />
-  <link rel="icon" href="favicon_light.png" media="(prefers-color-scheme: dark)" />
+  <link rel="icon" href="favicon_light.png" />
```
Comment (valentinoConti) [inline src/index.html:9]: This change it's wrong, on dark mode works all right:

dark

but on light mode works wrong:

light bad

and without this cange its working fine for both modes:

light good

Comment (raop155) [reply src/index.html:9]: I can leave as is, but George said that we should always use "light" version, we cant change the logo colors.

Comment (valentinoConti) [reply src/index.html:9]: Leave it as it is 👌🏼 

image

**Review by valentinoConti — APPROVED** [review]

File: src/index.html (hunk 2 above)
Comment (raop155) [reply src/index.html]: done!

**Pull request merged**